src/pages/hegel/reference/nothing/development.mdx

```diff
@@ -35,8 +35,9 @@ Equally, thinking of `nothing` in terms of negation or negativity is also inadeq
 
 The idea of `nothing` cannot but be considered through itself&mdash;in "simple equality with itself"&mdash;that is, without reference to anything else.
```

Comment:
Above you do suggest that one could think of nothing as the absence of being. Now you may be using that as a heuristic, but it might come across as a contradiction or as slightly confusing for the reader.

Reply:
Could you please expand(TM) how might it come across as a contradiction?

Reply:
I should have been more precise: on line 30 you say that: "one can say it is the absence of being". Then, on line 34 you say " thinking of `nothing` in terms of negation or negativity is also inadequate". This would be the potential contradiction. Isn't thinking of nothing as the absence of being, thinking of nothing in terms of negation or negativity? Isn't Nothing _as_ the absence of being, Nothing as the negativity of being?

Appreciate the TM by the way :)

Reply:
I don't think "absence of being" is an issue here. I see where you're coming from but "absence of being" I think of as one idea that cannot be broken down into "being" and "absence". It's rather a synonym for `nothing` rather than a negation of `being`. I also phrase it as "one can say..." which is meant to be a helper.

Reply:
I understand why you wrote what you wrote. I just think it's open to misunderstanding. But if you don't think that it is, feel free to resolve this.

Reply:
We can circle back to this if it does turn out to be inviting misunderstanding. Meanwhile, I look forward to your heterodox reading.